opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -50,6 +50,7 @@ pub(crate) mod meter;
 mod meter_provider;
 pub(crate) mod noop;
 pub(crate) mod periodic_reader;
+pub(crate) mod periodic_reader_with_own_thread;
```

Comment:
Would it make sense to add this behind a feature flag to indicate to end-users that it’s still experimental ?

Reply:
added